app/onboarding/page.tsx

```diff
@@ -0,0 +1,170 @@
+'use client';
+
+import React, { useState } from 'react';
+import { ProfileProvider, useProfile } from '@/utils/ProfileProvider';
+
+// Define the possible options for each question
+const states = ['Tennessee', 'Missouri'];
+const gardenTypes = ['Individual', 'Community', 'School'];
+const plotOptions = [
+  { label: 'yes', value: true },
+  { label: 'no', value: false },
+];
+//Interfaces and props to avoid typ errors on Lint
+interface StateSelectionProps {
+  selectedState: string;
+  setSelectedState: React.Dispatch<React.SetStateAction<string>>;
+}
+
+interface GardenTypeSelectionProps {
+  selectedGardenType: string;
+  setSelectedGardenType: React.Dispatch<React.SetStateAction<string>>;
+}
+
+interface PlotSelectionProps {
+  selectedPlot: boolean | null;
+  setSelectedPlot: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+// Select State
+const StateSelection: React.FC<StateSelectionProps> = ({
+  selectedState,
+  setSelectedState,
+}) => {
+  return (
+    <div>
+      <h2>Which state do you live in?</h2>
+      <select
+        value={selectedState}
+        onChange={e => setSelectedState(e.target.value)}
+      >
+        <option value="" disabled>
+          Select a state
+        </option>
+        {states.map(state => (
+          <option key={state} value={state}>
+            {state}
+          </option>
+        ))}
+      </select>
+    </div>
+  );
+};
+
+// Step 2: Select garden type
+
+const GardenTypeSelection: React.FC<GardenTypeSelectionProps> = ({
+  selectedGardenType,
+  setSelectedGardenType,
+}) => {
+  return (
+    <div>
+      <h2>What type of garden do you want to create?</h2>
+      {gardenTypes.map(type => (
+        <label key={type}>
+          <input
+            type="radio"
+            name="gardenType"
+            value={type}
+            checked={selectedGardenType === type}
+            onChange={e => setSelectedGardenType(e.target.value)}
+          />
+          {type}
+        </label>
+      ))}
+    </div>
+  );
+};
+
+// Step 3: Do you have a plot?
+const PlotSelection: React.FC<PlotSelectionProps> = ({
```

Comment:
we can probably replace all the `React.FC` types with regular function definitions, so like 
```
const func PlotSelection ({selectedPlot, setSelectedPlot}: PlotSelectionProps) {
...
}
```